app/models/membership.rb

```diff
@@ -12,6 +12,8 @@ class Membership < ActiveRecord::Base
 
   after_commit :update_relationships
 
+  scope :ambassador_organizations, -> { where(organization: Organization.ambassador).order(:created_at) }
```

Comment:
Oooh, nifty. Does this generate good sql?

Reply:
I think so? If I'm missing a better approach, lmk. This is the SQL that's generated:

```sql
SELECT
   "memberships".* 
FROM
   "memberships" 
WHERE
   "memberships"."deleted_at" IS NULL 
   AND "memberships"."organization_id" IN 
   (
      SELECT
         "organizations"."id" 
      FROM
         "organizations" 
      WHERE
         "organizations"."deleted_at" IS NULL 
         AND "organizations"."kind" = $1 
      ORDER BY
         "organizations"."name" ASC
   )
ORDER BY
   "memberships"."created_at" ASC 
```

An alternative:

```ruby
Membership.includes(:organization).where(organizations: {kind: 8})
```
```sql
SELECT
   COUNT(DISTINCT "memberships"."id") 
FROM
   "memberships" 
   LEFT OUTER JOIN
      "organizations" 
      ON "organizations"."id" = "memberships"."organization_id" 
      AND "organizations"."deleted_at" IS NULL 
WHERE
   "memberships"."deleted_at" IS NULL 
   AND "organizations"."kind" = $1 


SELECT
      "memberships"."id" AS t0_r0,
      "memberships"."organization_id" AS t0_r1,
      "memberships"."user_id" AS t0_r2,
      "memberships"."role" AS t0_r3,
      "memberships"."invited_email" AS t0_r4,
      "memberships"."created_at" AS t0_r5,
      "memberships"."updated_at" AS t0_r6,
      "memberships"."deleted_at" AS t0_r7,
      "organizations"."id" AS t1_r0,
      "organizations"."name" AS t1_r1,
      "organizations"."slug" AS t1_r2,
      "organizations"."available_invitation_count" AS t1_r3,
      "organizations"."created_at" AS t1_r4,
      "organizations"."updated_at" AS t1_r5,
      "organizations"."website" AS t1_r6,
      "organizations"."short_name" AS t1_r7,
      "organizations"."show_on_map" AS t1_r8,
      "organizations"."sent_invitation_count" AS t1_r9,
      "organizations"."deleted_at" AS t1_r10,
      "organizations"."is_suspended" AS t1_r11,
      "organizations"."auto_user_id" AS t1_r12,
      "organizations"."access_token" AS t1_r13,
      "organizations"."api_access_approved" AS t1_r14,
      "organizations"."approved" AS t1_r15,
      "organizations"."avatar" AS t1_r16,
      "organizations"."is_paid" AS t1_r17,
      "organizations"."lock_show_on_map" AS t1_r18,
      "organizations"."landing_html" AS t1_r19,
      "organizations"."paid_feature_slugs" AS t1_r20,
      "organizations"."parent_organization_id" AS t1_r21,
      "organizations"."kind" AS t1_r22,
      "organizations"."ascend_name" AS t1_r23,
      "organizations"."registration_field_labels" AS t1_r24 
   FROM
      "memberships" 
      LEFT OUTER JOIN
         "organizations" 
         ON "organizations"."id" = "memberships"."organization_id" 
         AND "organizations"."deleted_at" IS NULL 
   WHERE
      "memberships"."deleted_at" IS NULL 
      AND "organizations"."kind" = $1 
```

Reply:
Cool, I like the readability of your initial approach